client/src/components/LeafletOsmMap/SavedPlaces.js

```diff
@@ -0,0 +1,159 @@
+import {h, Component} from "preact";
+import {makeRequest} from "../../js/server-requests-utils";
+
+/**
+ * Leaflet related imports: leaflet, pouchdb module, and routing machine module
+ */
+import '../../../node_modules/leaflet/dist/leaflet.css';
+import '../../../node_modules/leaflet-routing-machine/dist/leaflet-routing-machine.css';
+import L from '../../js/leaflet-tileLayer-pouchdb-cached';
+
+/* Set all settings for saved place icon
+Waiting for new icon for favorites
+Full list of options: http://leafletjs.com/reference-1.3.0.html#icon
+ */
+const FAV_MARKER_ICON = L.icon({
+  iconUrl: '../../assets/icons/leaflet/marker-icon-fav-2x.png',
+  className: 'favorites',
+  iconSize: [25, 41], //native aspect ratio: [28, 41]
+});
+
+
+const CURRENT_USER = makeRequest('GET', 'user')
+  .catch(err=>{
+    //user not logged in
+  });
+ 
+ 
+
+export default class SavedPlaces extends Component {
+  constructor(props) {
+    super(props)
+    const {user, map} = props;
+
+    fetchAndDropUserPins(user, map);
+  }
+
+  render() {
+    //This shouldn't render much (if anything)
+    return ( null  );
+  }
+
+}
+ 
+
+/**
+ * Exported Functions
+ */
+
+//Get and drop pins from any user on any map
+const fetchAndDropUserPins = (userPromised = CURRENT_USER, mapObj = undefined) => {
+
+  userPromised
```

Comment:
This isn't needed...the backend server api endpoint for `{GET} /search/savedpins` will return all saved points that match given user id with valid authenticated token. No need to get user id before making request to this end point.